conformance_tests/sysman/test_sysman_device/src/test_sysman_device_helper.cpp

```diff
@@ -51,6 +85,31 @@ void get_ze_root_uuids(std::vector<ze_device_handle_t> ze_devices,
   }
 }
 
+void get_ze_device_uuids(std::vector<ze_device_handle_t> ze_devices,
+                         std::vector<ze_device_uuid_t> &ze_device_uuids,
+                         char *device_hierarchy) {
+  for (const auto &ze_device : ze_devices) {
+    auto ze_device_properties = lzt::get_device_properties(ze_device);
+    auto ze_device_uuid = ze_device_properties.uuid;
+    ze_device_uuids.push_back(ze_device_uuid);
+  }
+}
+
+void compare_core_and_sysman_uuid(std::vector<ze_device_uuid_t> core_uuids,
+                                  std::vector<zes_uuid_t> sysman_uuids) {
+  for (const auto &core_uuid : core_uuids) {
+    bool core_and_sysman_uuid_equal = false;
+    for (const auto &sysman_uuid : sysman_uuids) {
+      if (memcmp(core_uuid.id, sysman_uuid.id, ZE_MAX_DEVICE_UUID_SIZE) ==
```

Comment:
Can you compare with integer instead of boolean ?

Reply:
Modified compare logic